typescript/infra/src/utils/utils.ts

```diff
@@ -213,6 +213,14 @@ export function assertRole(roleStr: string) {
   return role;
 }
 
+export function assertFundableRole(roleStr: string): FundableRole {
```

Comment:
maybe enum?

Reply:
like 
```
export enum FundableRole {
  Relayer = 'Relayer',
  Kathy = 'Kathy',
}```

doesn't make too much of a difference

Reply:
not a fan of the enum way, just adds another place to update things in the future if needed

imo makes more sense the way it is, re-using existing Roles enum wherever poss

Reply:
I think you could reuse the existing enum but I digress